cl/search/views.py

```diff
@@ -41,6 +42,54 @@ def check_pagination_depth(page_number):
     return False
 
 
+def get_mlt_query(cd, facet, solr_url, seed_pk, filter_query):
```

Comment:
I think this should probably go in `search_utils.py`. Lots of other random solr querying junk like this in there.